ExtraDry/Sample.Shared/Entitites/Company.cs

```diff
@@ -47,9 +48,9 @@ public class Company {
     [Rules(RuleAction.IgnoreDefaults)]
     public string Description { get; set; } = string.Empty;
 
-    [Display]
+    [Display(Name = "Primary Sector", ShortName = "Sector")]
     [Rules(RuleAction.Link)]
-    [JsonConverter(typeof(SectorInfoJsonConverter))]
+    [JsonConverter(typeof(ResourceReferenceConverter<Sector>))]
```

Comment:
Yep, SectorInfo as a separate object was a previous concept and needed to go.  Does this feel cleaner when you're replacing it?

Reply:
It feels a lot better with a common converter.  I could imagine the number of converts growing out of control if not for the change.